packages/k4actstracking/package.py

```diff
@@ -19,7 +19,7 @@ class K4actstracking(CMakePackage, Key4hepPackage):
     depends_on("root")
     depends_on("edm4hep")
     depends_on("k4fwcore")
-    depends_on("acts-dd4hep")
+    depends_on("acts-dd4hep", when="^acts@:19")
```

Comment:
```suggestion
```
We may as well delete since I guess no one is going to build ACTS < v20. The `acts-dd4hep` repository has been archived for more than two years now.